app/common/alerts.py

```diff
@@ -18,6 +19,7 @@ def send(email, reference):
                                                      template_id=settings.NOTIFICATION_TEMPLATE_ID,
                                                      personalisation=None,
                                                      reference=reference)
+        logger.info('Sent secure message email notification', user=g.user.user_uuid)
```

Comment:
Should this be named user_uuid to keep consistent and be more specific in the logs?